app/views/RoomView/RightButtons.tsx

```diff
@@ -157,6 +181,82 @@ class RightButtonsContainer extends Component<IRightButtonsProps, IRigthButtonsS
 		}
 	};
 
+	returnLivechat = () => {
+		const { rid } = this.props;
+		showConfirmationAlert({
+			message: i18n.t('Would_you_like_to_return_the_inquiry'),
+			confirmationText: i18n.t('Yes'),
+			onPress: async () => {
+				try {
+					await RocketChat.returnLivechat(rid);
+				} catch (e: any) {
+					showErrorAlert(e.reason, i18n.t('Oops'));
+				}
+			}
+		});
+	};
+
+	placeOnHoldLivechat = () => {
+		const { navigation, rid } = this.props;
+		showConfirmationAlert({
+			title: i18n.t('Are_you_sure_question_mark'),
+			message: i18n.t('Would_like_to_place_on_hold'),
+			confirmationText: i18n.t('Yes'),
+			onPress: async () => {
+				try {
+					await RocketChat.onHoldLivechat(rid);
+					navigation.navigate('RoomsListView');
+				} catch (e: any) {
+					showErrorAlert(e.data?.error, i18n.t('Oops'));
+				}
+			}
+		});
+	};
+
+	closeLivechat = () => {
+		const { dispatch, rid } = this.props;
+		dispatch(closeRoom(rid));
+	};
```

Comment:
I'm wondering if it's better to extract those as well to `RoomView` to keep it DRY or if it's better to leave them on each component (`RoomActionsView` | `RightButtons`)? Another possible solution seems possible (e.g. move them to a separate file, maybe?), but I'm open to anything that the reviewer agrees to 😎